tests/runtests.py

```diff
@@ -40,7 +40,17 @@
 except OSError as err:
     print('Failed to change directory to salt\'s source: {0}'.format(err))
 
-REQUIRED_OPEN_FILES = 3072
+# Soft and hard limits on max open filehandles
+MAX_OPEN_FILES = {
+    'integration': {
+        'soft_limit': 3072,
+        'hard_limit': 4096,
+    },
+    'unit': {
+        'soft_limit': 1024,
+        'hard_limit': 2048,
```

Comment:
Hmmm, OK. Let's keep a close eye on this. I don't want to mask something that might be an issue with Salt itself by raising it here.